src/main/java/org/jenkins_ci/plugins/run_condition/core/CauseCondition.java

```diff
@@ -66,15 +68,17 @@ public boolean isCausedBy(String className) {
 
         // if XTrigger plugin is installed:
         // file change
-        FS_CAUSE("org.jenkinsci.plugins.fstrigger.FSTriggerCause", "FSTrigger"),
+        FS_CAUSE(XTriggerCause.class, "FSTrigger"),
```

Comment:
why did you change this to use the class instead of the class name as a String? now the plugin has a dependency to the `xtrigger` plugin and everyone using this plugin needs to install that plugin too. Please change this back to use the full qualified name of the class as a String